docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -70,6 +70,18 @@ Access
 - Access to Amazon `S3`_ with an `Access Key ID and Secret Access Key <https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html>`_.
   It is possible that you will have an account but not the permissions to create an Access Key ID and Secret Access Key.
 
+  - Check whether you have an existing access key by running the ``list-access-keys`` command:
+
+   .. prompt:: bash $
+
+      aws iam list-access-keys
+
+  - Create a new access key and secret token if necessary:
+
+   .. prompt:: bash $
+
+      aws iam create-access-key
```

Comment:
If I run either of these commands with a newly installed `awscli` on a new machine (in this case a Docker container) I get:

```
(flocker)root@3c687e22db87:/# aws iam list-access-keys
Unable to locate credentials. You can configure credentials by running "aws configure".
(flocker)root@3c687e22db87:/# aws iam create-access-key
Unable to locate credentials. You can configure credentials by running "aws configure".
```

In what situation if this useful?